<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>site_categoriserDockWidgetBase</class>
 <widget class="QDockWidget" name="site_categoriserDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>site categoriser</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="main_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="database_label">
           <property name="text">
            <string>not connected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="connect_button">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Connect to database.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Connect...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="prepare_database_button">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;First time setup for database.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Prepare Database</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Junct/Cross</string>
          </attribute>
          <attribute name="toolTip">
           <string>curvature and gradient table. Shouldn't need to edit this.</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>add:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cat_combo">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Category to add.&lt;/p&gt;&lt;p&gt;Q1=minor junction,&lt;/p&gt;&lt;p&gt;Q2=major junction,&lt;/p&gt;&lt;p&gt;K=crossing&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <item>
                <property name="text">
                 <string>K</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Q1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Q2</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="add_button">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add junction or crossing with selected section,chainage and category.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>ok</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="jc_view">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Junction and Crossings.  &amp;lt;50m before section should be added with negative chainage. &lt;/p&gt;&lt;p&gt; &amp;lt;50m after section with chainage&amp;gt;section length.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
